### ISSUE TITLE
Coordinator node log overrun by "skipping doNext()..."

### ISSUE THREAD

**Issue by erikdubbelboer** [druid-io/druid#4609]
We are running a `0.10.1-rc3-SNAPSHOT` build. I just started a new historical node that needs to load 3406 segments (around 478GB). Now the coordinator log is completely being overrun by the following message:
```
Server[/druid/loadQueue/druid-historical-1wwd:8083] skipping doNext() because something is currently loading[requests_2017-06-21T14:00:00.000Z_2017-06-21T15:00:00.000Z_2017-06-21T14:00:00.000Z_1].
```
This message is being printed hundreds of times every couple of seconds.

Our coordinator config is quite standard I think:
```
druid.host=druid-eu-coordinator-1
druid.port=8081
druid.service=druid/coordinator

# The coordinator begins assignment operations after the start delay.
# We override the default here to start things up faster for examples.
# In production you should use PT5M or PT10M
druid.coordinator.startDelay=PT5M

druid.coordinator.merge.on=false
druid.coordinator.conversion.on=true

druid.coordinator.load.timeout=PT5M

druid.coordinator.kill.on=true
druid.coordinator.kill.durationToRetain=P1D
druid.coordinator.kill.maxSegments=1024
```

This is the `log4j2.xml` config:
```xml
<?xml version="1.0" encoding="UTF-8" ?>
<Configuration status="WARN" shutdownHook="disable">
  <Appenders>
    <Console name="Console" target="SYSTEM_OUT">
      <PatternLayout pattern="%d{ISO8601} %p [%t] %c - %m%n"/>
    </Console>

    <Syslog name="RFC5424" format="RFC5424" host="localhost" port="514"
      protocol="TCP" appName="druid" mdcId="mdc" includeMDC="true"
      facility="LOCAL0" enterpriseNumber="18060" newLine="true"
      messageId="Audit" id="App">
      <ThresholdFilter level="ERROR" onMatch="ACCEPT" onMismatch="DENY"/>
    </Syslog>

    <RollingFile name="RollingFile" fileName="/var/log/druid/coordinator.log"
      filePattern="/var/log/druid/coordinator-%d{yyyy-MM-dd}-%i.log.gz">
      <PatternLayout>
        %d{ISO8601} %p [%t] %c - %m%n
      </PatternLayout>
      <Policies>
        <TimeBasedTriggeringPolicy />
        <SizeBasedTriggeringPolicy size="250 MB"/>
      </Policies>
      <DefaultRolloverStrategy max="10"/>
    </RollingFile>
  </Appenders>
  <Loggers>
    <Root level="info">
      <AppenderRef ref="Console"/>
      <AppenderRef ref="RFC5424"/>
      <AppenderRef ref="RollingFile"/>
    </Root>
  </Loggers>
</Configuration>
```

**Comment by gianm** [comment]
Is this new behavior in 0.10.1? Wondering if it needs to be a 0.10.1 release blocker or not.

**Comment by erikdubbelboer** [comment]
I'm not sure, we only started adding nodes after we upgraded to 0.10.1 and I don't really want to downgrade.

**Comment by jon-wei** [comment]
Thanks for the report @erikdubbelboer!

This is related to the LoadQueuePeon changes from https://github.com/druid-io/druid/pull/3736.

These logs would be generated every 50ms for each historical node that's currently loading segments.

I'm moving this issue to 0.11.0 since the related changes were introduced in 0.10.0 and the excessive logs, while troublesome, aren't a critical blocking failure.

@niketh Are you able to take a look at this issue?

**Comment by niketh** [comment]
@jon-wei Looking at it. Thanks for pointing out.

**Comment by jon-wei** [comment]
fixed by #4680
